core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.html

```diff
@@ -26,13 +36,17 @@
       <strong>Please translate the given {{ activeContentType }} in {{ languageDescription }}:</strong>
       <a class="oppia-skip-button" *ngIf="moreAvailable" (click)="skipActiveTranslation()">skip</a>
     </div>
-    <div *ngIf="!loadingData && !uploadingTranslation" class="oppia-text-to-translate-container">
+    <div *ngIf="!loadingData && !uploadingTranslation"
+         class="oppia-text-to-translate-container"
+         [ngClass]="isContentExpanded ? 'oppia-expanded' : 'oppia-contracted'"
+         #contentContainer>
       <!-- textToTranslate contains the html content of an exploration state -->
       <oppia-rte-output-display *ngIf="activeDataFormat === 'html' || activeDataFormat === 'unicode'"
                                 [ngClass]="{'oppia-rte-editor-focused': isCopyModeActive()}"
                                 (click)="onContentClick($event)"
                                 [rteString]="textToTranslate"
-                                class="oppia-rte-editor">
+                                class="oppia-rte-editor"
+                                #contentPanel>
```

Comment:
What does this do exactly?

Reply:
The `#contentPanel` is a template reference variable used to fetch the 'height' of this component in the .ts file with the help of `@ViewChild`.